closed/autoconf/custom-hook.m4

```diff
@@ -377,7 +377,7 @@ AC_DEFUN([OPENJ9_CONFIGURE_CRIU_SUPPORT],
     AC_MSG_RESULT([no (explicitly disabled)])
   elif test "x$enable_criu_support" = x ; then
     case "$OPENJ9_PLATFORM_CODE" in
-      xa64|xz64)
+      xa64|xz64|xr64)
```

Comment:
We aim to keep lists like this sorted (when it otherwise doesn't matter) making it easier for humans to comprehend.